A change in Qt 5.11_beta3 leads to errors

Arch GNU/Linux, `bionic` branch
```
CMake Error at src/qdbus-stubs/CMakeLists.txt:167 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```
According to [the official Qt documentation](https://doc.qt.io/qt-5/cmake-manual.html) `qt5_use_modules` macro is obsolete 



Same as https://github.com/ubports/ubuntu-download-manager/issues/4

--- a/src/connectivity-api/connectivity-qml/CMakeLists.txt
+++ b/src/connectivity-api/connectivity-qml/CMakeLists.txt
@@ -16,13 +16,8 @@ add_library(connectivity-qml SHARED
 target_link_libraries(
   connectivity-qml
   ${CONNECTIVITY_QT_LIB_TARGET}
+  Qt5::Qml
 )
-
-
-qt5_use_modules(connectivity-qml
-  Qml
-)
-
 
 add_qmlplugin(
   Ubuntu.Connectivity

--- a/src/connectivity-api/connectivity-qt/CMakeLists.txt
+++ b/src/connectivity-api/connectivity-qt/CMakeLists.txt
@@ -91,14 +91,10 @@ add_library(
     ${PUBLIC_HEADERS} # public headers have to be included here for Q_DECL_EXPORT to work properly
 )
 
-qt5_use_modules(
-    ${CONNECTIVITY_QT_LIB_TARGET}
-    Core
-    DBus
-)
-
 target_link_libraries(
     ${CONNECTIVITY_QT_LIB_TARGET}
+    Qt5::Core
+    Qt5::DBus
     qdbus-stubs
     util
 )

--- a/src/qdbus-stubs/CMakeLists.txt
+++ b/src/qdbus-stubs/CMakeLists.txt
@@ -164,7 +164,7 @@ add_library(
     ${CONNECTIVITY_BACKEND_SRC}
 )
 
-qt5_use_modules(
+target_link_libraries(
     qdbus-stubs
-    DBus
+    Qt5::DBus
 )

--- a/tests/integration/CMakeLists.txt
+++ b/tests/integration/CMakeLists.txt
@@ -33,15 +33,11 @@ add_executable(
     ${INTEGRATION_TESTS_SRC}
 )
 
-qt5_use_modules(
-    integration-tests
-    Core
-    DBus
-    Test
-)
-
 target_link_libraries(
     integration-tests
+    Qt5::Core
+    Qt5::DBus
+    Qt5::Test
     test-utils
     ${CONNECTIVITY_QT_LIB_TARGET}
     ${TEST_DEPENDENCIES_LDFLAGS}

--- a/tests/unit/CMakeLists.txt
+++ b/tests/unit/CMakeLists.txt
@@ -20,13 +20,9 @@ add_executable(
     secret-agent/secret-agent-main.cpp
 )
 
-qt5_use_modules(
-    secret-agent-test-bin
-    Core
-)
-
 target_link_libraries(
     secret-agent-test-bin
+    Qt5::Core
     agent-static
 )
 
@@ -62,15 +58,11 @@ add_executable(
     ${UNIT_TESTS_SRC}
 )
 
-qt5_use_modules(
-    unit-tests
-    Core
-    DBus
-    Test
-)
-
 target_link_libraries(
     unit-tests
+    Qt5::Core
+    Qt5::DBus
+    Qt5::Test
     test-utils
     indicator-network-service-static
     ${TEST_DEPENDENCIES_LDFLAGS}

